gma/integration_test/src/integration_test.cc

```diff
@@ -821,13 +826,15 @@ TEST_F(FirebaseGmaTest, TestInterstitialAdLoad) {
   EXPECT_FALSE(result_ptr->response_info().ToString().empty());
 
   delete interstitial;
+
+  FLAKY_TEST_SECTION_END();
 }
 
 TEST_F(FirebaseGmaTest, TestRewardedAdLoad) {
   SKIP_TEST_ON_DESKTOP;
+  SKIP_TEST_ON_SIMULATOR;
 
-  // Note: while showing an ad requires user interaction (below),
-  // we test that we can simply load an ad first.
+  FLAKY_TEST_SECTION_BEGIN();
```

Comment:
It would be good, here and elsewhere, to explain why this may be flaky. i.e. that the backend may throttle us or not be able to fill the ad.

Reply:
Done.